im-gespraech-buber-gesellschaft.csl

```diff
@@ -0,0 +1,651 @@
+  </macro>
+  <macro name="date">
+    <choose>
+      <if match="any" variable="issued">
+        <date form="text" variable="issued"/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="collection-editor">
+    <names variable="collection-editor" prefix="Hrsg. von ">
+      <name delimiter="/" delimiter-precedes-last="always"/>
+    </names>
+  </macro>
+  <macro name="volume">
+    <group suffix=".">
+      <text term="volume" form="short" suffix=" "/>
+      <text variable="volume"/>
+    </group>
+  </macro>
+  <macro name="Band aus Reihe">
+    <choose>
+      <if match="any" variable="title">
+        <group>
+          <text macro="editor" prefix=" "/>
+          <group prefix=" (" suffix=") ">
+            <group>
+              <text variable="collection-title"/>
+              <choose>
+                <if match="any" variable="collection-editor">
+                  <text value=". "/>
+                </if>
+              </choose>
+            </group>
+            <text macro="collection-editor"/>
+            <text variable="volume" prefix=", Bd. "/>
+          </group>
+        </group>
+      </if>
+      <if match="none" variable="title">
+        <group>
+          <group>
+            <text variable="collection-title" font-style="italic"/>
+            <choose>
+              <if match="any" variable="collection-editor">
+                <text value=" ."/>
+              </if>
+            </choose>
+          </group>
+          <text macro="collection-editor"/>
+          <text macro="volume" prefix=", "/>
+          <text macro="editor"/>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="container-title">
+    <choose>
+      <if match="any" variable="container-title">
+        <text macro="editor" prefix=" "/>
+        <group prefix=" (" suffix=") ">
+          <group>
+            <text variable="collection-title"/>
+            <choose>
+              <if match="any" variable="collection-editor">
+                <text value=". "/>
+              </if>
+            </choose>
+          </group>
+          <text macro="collection-editor"/>
+          <text variable="volume" prefix=", Bd. "/>
+        </group>
+      </if>
+      <if match="none" variable="container-title">
+        <group>
+          <text variable="collection-title" font-style="italic" prefix=" " suffix="."/>
+          <text macro="collection-editor" prefix=" "/>
+          <text macro="volume" prefix=" "/>
+          <text macro="editor" font-style="normal" prefix=" "/>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="edition-number">
+    <choose>
+      <if match="any" variable="edition">
+        <text variable="edition" prefix=" " suffix=". "/>
+        <label plural="never" variable="edition"/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="container-author">
+    <choose>
+      <if match="any" variable="container-author">
+        <text value="Ders.: "/>
```

Comment:
I don't think this'll work, CSL can't handle ders.
Container author would typically be the author of a book where you're citing e.g. the introduction by someone else. E.g. in Nabokov, Vladimir, "Einleitung," in Kafka, Franz, *Die Metamorphose,* ...
Kafka is the container-author.